sharedutils/sharedutils.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    `java-library`
+}
+
+description = "Common utilities for the add-ons."
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    compileOnly("org.zaproxy:zap:2.7.0")
+
+    api("junit:junit:4.11")
+    api("log4j:log4j:1.2.17")
+    api("org.hamcrest:hamcrest-library:1.3")
+    api("org.mockito:mockito-core:3.1.0")
```

Comment:
```kts
    val zap = "org.zaproxy:zap:2.7.0"
    compileOnly(zap)

    testImplementation(project(":testutils"))
    testImplementation(zap)
```
otherwise it would expose the test dependencies to the other add-ons.